src/hir/translator.rs

```diff
@@ -74,6 +76,11 @@ impl<'a> TranslatorI<'a> {
     }
 }
 
+#[derive(Debug, Deserialize)]
+struct BulloakToml {
+    no_skip: bool,
+}
```

Comment:
I would actually prefer we don't include the work for adding a `bulloak.toml` in this PR. I want to be conservative as to the process of adding new features, and adding a config file is quite a jump IMO. I'm not even convinced that's the way forward as you can see in the relevant [issue](https://github.com/alexfertel/bulloak/issues/38).

Also, I'd want to keep this PR isolated from anything else.

Reply:
This should be a cmd flag. Something like `with_vm_skip`, i.e. it should be positive (`no_skip` is negative), with a `short = 'c'` maybe. It should default to `false` since we don't want to break people who upgrade to this version, effectively making it opt-in. Does this make sense?

Reply:
> dep injection super tidy,

Sorry about that, it definitely needs improvement.

> we should then use cfg struct

yes, that makes sense 👍

If possible, I think it's best to actually do that change in a separate PR from the changes required for fuzzing.